content/applications/websites/ecommerce/checkout_payment_shipping/ewallet_giftcard.rst

```diff
@@ -0,0 +1,122 @@
+======================
+eWallet and gift cards
+======================
+
+You can enable **eWallets** and **gift cards** for your customers on your eCommerce, which they can
+use to for their online and in-store shopping. To allow eWallets and gift cards for your eCommerce,
+head to :menuselection:`Website app --> eCommerce` and in the :guilabel:`Shop - Products` category,
+enable :guilabel:`Discounts, Loyalty & Gift Card`.
+
+eWallet
+=======
+
+eWallet allows customers to save their credits on their online account and use these credits as a
+payment method when buying items on your online store. As multiple redeems for the same order are
+not possible in Odoo, eWallets can also be used by customers to centralize multiple :ref:`gift cards
+<ewallet_gift/gift-cards>` and accumulate the credits.
+
+Before creating an eWallet program, it is necessary to first create an eWallet **top-up** product.
+To do so, go to :menuselection:`Website app --> eCommerce --> Products` and :guilabel:`Create` a new
+product. On the product template, configure the product as such:
+
+- :guilabel:`Can be sold`: enabled;
+- :guilabel:`Product Type`: set as :guilabel:`Service`;
+- :guilabel:`Invoicing Policy`: set as :guilabel:`Prepaid/Fixed Price`;
+- :guilabel:`Create on Order`: select :guilabel:`Nothing`;
+- :guilabel:`Sales Price`: type in the **amount** of the top-up.
+
+.. note::
+   If you wish to have eWallet **top-ups** of different amounts, repeat the process and modify the
+   :guilabel:`Sales Price` accordingly.
+
+Once the **top-up** is created, go to :menuselection:`Website app --> eCommerce --> Gift Cards &
+eWallet` to :guilabel:`Create` an eWallet **program**. The following configuration options are
+available:
+
+- :guilabel:`Program Name`: enter the name you want;
+- :guilabel:`Program Type`: select :guilabel:`eWallet`;
+- :guilabel:`eWallet Products`: select the eWallet **top-up(s)** created earlier;
+- :guilabel:`Email template`: the email template used for the email sent to the customer, you can
+  create a template by clicking the field and selecting :guilabel:`Search more`;
+- :guilabel:`Currency`: select the currency you wish to use for that eWallet program;
+- :guilabel:`Company`: the company in which the program is valid and available;
+- :guilabel:`Available On`: the different stores where the program is valid and available;
+- :guilabel:`Website`: the website on which the program is valid and available, leave empty for all;
+- :guilabel:`Point of Sale`: :abbr:`PoS (Point of Sale)` in which the program is valid and
+  available, leave empty for all.
+
+.. image:: ewallet_gift/ewallet-configuration.png
```

Comment:
I don't have the same image 
![image](https://user-images.githubusercontent.com/85342080/204999185-085854e0-26cf-4340-8680-870f4d65bf04.png)